pyro/distributions/planar.py

```diff
@@ -52,24 +52,28 @@ class PlanarFlow(Transform):
     codomain = constraints.real
 
     def __init__(self, input_dim):
-        super(PlanarFlow, self).__init__()
+        Transform.__init__(self)
+        nn.Module.__init__(self)
```

Comment:
It is really gross to call multiple parent constructors. Let's clean this up soon, either in this PR or in a follow-up PR.